Magento/ruleset.xml

```diff
@@ -5,202 +5,402 @@
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.PHP.LowerCaseConstant">
+        <severity>6</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.PHP.LowerCaseKeyword">
+        <severity>6</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.WhiteSpace.DisallowTabIndent">
+        <severity>6</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.WhiteSpace.ScopeIndent">
+        <severity>6</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Whitespace">
+    <rule ref="Magento.Whitespace.EmptyLineMissed">
         <severity>6</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Magento.Whitespace.MultipleEmptyLines">
+        <severity>6</severity>
+        <type>warning</type>
     </rule>
     <rule ref="PEAR.ControlStructures.ControlSignature">
         <severity>6</severity>
         <type>warning</type>
     </rule>
-    <rule ref="PSR2">
-        <severity>8</severity>
+    <rule ref="PSR1.Files.SideEffects">
+        <severity>6</severity>
         <type>warning</type>
-        <exclude name="Squiz.WhiteSpace.ControlStructureSpacing.NoLineAfterClose"/>
-        <exclude name="Generic.Files.LineLength"/>
     </rule>
-    <rule ref="PSR1.Classes.ClassDeclaration">
-        <severity>10</severity>
-        <type>error</type>
+    <rule ref="PSR1.Methods.CamelCapsMethodName">
+        <severity>6</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="PSR2.Files.ClosingTag">
-        <severity>10</severity>
-        <type>error</type>
-        <exclude-pattern>*.phtml</exclude-pattern>
+    <rule ref="PSR2.Classes.ClassDeclaration">
+        <severity>6</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Squiz.Commenting.DocCommentAlignment">
+    <rule ref="PSR2.Classes.PropertyDeclaration">
         <severity>6</severity>
         <type>warning</type>
     </rule>
-    <rule ref="Squiz.Operators.IncrementDecrementUsage">
+    <rule ref="PSR2.ControlStructures.ControlStructureSpacing">
         <severity>6</severity>
         <type>warning</type>
     </rule>
-    <rule ref="Squiz.PHP.CommentedOutCode">
+    <rule ref="PSR2.ControlStructures.ElseIfDeclaration">
         <severity>6</severity>
-        <properties>
-            <property name="maxPercentage" value="70"/>
-        </properties>
+        <type>warning</type>
     </rule>
-    <rule ref="Squiz.PHP.Eval.Discouraged">
-        <severity>10</severity>
-        <type>error</type>
+    <rule ref="PSR2.ControlStructures.SwitchDeclaration">
+        <severity>6</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Squiz.PHP.GlobalKeyword">
-        <severity>8</severity>
+    <rule ref="PSR2.Files.EndFileNewline">
+        <severity>6</severity>
         <type>warning</type>
     </rule>
-    <rule ref="Squiz.PHP.NonExecutableCode">
+    <rule ref="PSR2.Methods.FunctionCallSignature">
```

Comment:
Revising PSR rules severities is not a good idea at all.

Reply:
Please provide more details why it is not a good idea.